packages/ariakit/src/combobox/__examples__/combobox-highlighted-text/style.css

```
@@ -0,0 +1,9 @@
+@import "../combobox/style.css";
+
+.combobox-item [data-user-value] {
+  @apply
+    font-semibold
+    text-primary-1
+    dark:text-primary-1-dark
+  ;
+}
```

It's hard to notice the highlighted part with those styles. Maybe something like this would work better?

```suggestion
.combobox-item [data-user-value] {
  @apply
    bg-warn-2
    text-warn-2
    dark:bg-warn-2
    dark:text-warn-2
  ;
}
```

---

Done, I resolve this